.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+> Note: This is the default pull request template.
+
+## What is the current behavior?
+> * Describe the current behavior of the application or program.
+> * Please mention any related issue numbers.
+
+## What's the issue?
+> * Explain in detail why this behavior is an issue - and why this should be changed.
+
+## How have you changed the behavior?
+> * Describe your changes in detail for the maintainer.
+> * Explain the solution you have provided, and *how* it fixes the issue case.
+> * Specify if this is a *breaking change*.
+
+## How was this change tested?
+> * Were unit test cases or end-to-end test cases included in this fix, or was only manual testing applicable?
+> * Please provide a detailed description of test cases included or updated.
+> * Does this change introduce any new warnings or binding errors? If so, make sure to update the relevant documentation and reference this pull request.
+
+---- 
+
+## Requirements
+
+> Your pull request must pass these requirements in order to be accepted. You can delete this section upon submitting your pull request.
+
+* [ ] My pull request title is prefixed with the directory and subdirectory it affects.
+* [ ] My pull request contains a single change or feature.
+* [ ] My commits are clearly labeled with one of the following: **feat | bug | fix | build | perf | task**
```

Comment:
missing period?

Reply:
I wouldn't add a period to the end of this list.